app/models/group.rb

```diff
@@ -414,6 +414,16 @@ def bulk_add(user_ids)
       if self.title.present?
         User.where(id: user_ids).update_all(title: self.title)
       end
+
+      if self.grant_trust_level.present?
+        User.where(id: user_ids).find_each do |user|
```

Comment:
Or like this? `User.where(id: user_ids).where("trust_level < ?", grant_trust_level).update_all(trust_level: grant_trust_level, trust_level_locked: true)`